src/Microsoft.Health.Dicom.Core.UnitTests/Extensions/DicomDatasetExtensionsTests.cs

```diff
@@ -54,14 +54,6 @@ public void GivenAValidDicomDateValue_WhenGetStringDateAsDateTimeIsCalled_ThenCo
                 _dicomDataset.GetStringDateAsDateTime(DicomTag.StudyDate));
         }
 
-        [Fact]
-        public void GivenAnInvalidDicomDateValue_WhenGetStringDateAsDateTimeIsCalled_ThenNullShouldBeReturned()
-        {
```

Comment:
Why was this test removed? is the behavior changed? is so does that affect our code?

Reply:
Sorry, fo-dicom updated and was throwing a validation error. I added back in the test case and disabled validation for it.